site/content/docs/5.2/customize/css-variables.md

```diff
@@ -28,6 +32,11 @@ Here are the variables we include (note that the `:root` is required) that can b
 {{< /root.inline >}}
 ```
 
+### Dark mode
+
+These variables are scoped to our built-in dark mode.
```

Comment:
Tried something to create this list (and to make work again the previous one) in https://github.com/twbs/bootstrap/pull/35857/commits/5031d416d58878c57473e0438042e48d1ff84014. I let you check that :)